### PR TITLE
Minor updates to the Occupational Corruption Device

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

This PR makes a few changes to the Occupational Corruption Device, the job-specific traitor item that allows you to summon a bureaucratic error event. Largely untouched for years, even across the change from standard to progression traitor.

It now triggers directly from the uplink, which means no more self-deleting remote. Now, it functions like the comms blackout/grid check "viruses".

This, however, led me to a major problem -- No remote means there is no longer an "Occupational Corruption Device", abandoning the pun intended by the original author.

To remedy this, the Occupational Corruption Device is now the "Organic Capital Disturbance virus". Different words, same stupid acronym. (The description is a bit more direct now too).

image

The event running has been moved to the force_event() helper, which I must have missed in #74904.

There is now a paltry 5 minute progression timer behind it, like the clown car.
## Why It's Good For The Game

Brings this traitor item more mechanically/codeilly in line with the other "buy a random event" options.

This is an item that's value scales with how early into the round it is used, but I feel like having a minor grace period would be fair to players who miss roundstart. 
## Changelog
:cl:
balance: The Occupational Corruption Device (now the "Organic Capital Disturbance" virus) now triggers directly from the uplink, rather than spawning a remote.
balance: The OCD now has a very small progression requirement behind it.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
would you kindly spell out what "OCD" stands for in at least one of your changelog entries - it's one of my pet peeves when people don't elaborate on what their acronym means. thanks

File: code/modules/uplink/uplink_items/job.dm [hunk 1]
```diff
@@ -16,14 +16,20 @@
...
+	desc = "Randomizes job positions presented to new hires. May lead to too many/too few security officers and/or clowns. Single use."
+	item = /obj/effect/gibspawner/generic
+	surplus = 0
 	limited_stock = 1
-	item = /obj/item/devices/bureaucratic_error_remote
+	cost = 2
+	progression_minimum = 5 MINUTES
+	restricted = TRUE
 	restricted_roles = list(JOB_HEAD_OF_PERSONNEL, JOB_QUARTERMASTER)
-	surplus = 5
```
Comment (san7890) [inline code/modules/uplink/uplink_items/job.dm:26]: why'd you nuke the surplus value to 0? i don't think it's a bad change and this is already a Balance PR but why'd you do it

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items/job.dm (hunk 1 above)
Comment (san7890) [reply code/modules/uplink/uplink_items/job.dm:26]: ah okay then

File: code/modules/uplink/uplink_items/job.dm [hunk 2]
```diff
@@ -16,14 +16,20 @@
 	limited_stock = 1 //please don't spam deadchat
 	surplus = 5
 
-/datum/uplink_item/role_restricted/bureaucratic_error_remote
-	name = "Organic Resources Disturbance Inducer"
-	desc = "A device that raises hell in organic resources indirectly. Single use."
-	cost = 2
+/datum/uplink_item/role_restricted/bureaucratic_error
+	name = "Organic Capital Disturbance virus"
```
Comment (JohnFulpWillard) [inline code/modules/uplink/uplink_items/job.dm:20]:
```suggestion
	name = "Organic Capital Disturbance Virus"
```
I think it looks bad otherwise

**Pull request merged**